packages/ui/src/components/Visualization/Canvas/CanvasSideBar.tsx

```diff
@@ -19,17 +19,30 @@ export const CanvasSideBar: FunctionComponent<CanvasSideBarProps> = (props) => {
      */
     <TopologySideBar show={props.selectedNode !== undefined}>
       <Card>
-        <CardTitle>
-          {props.selectedNode?.label}{' '}
-          <Button
-            variant="plain"
-            icon={<TimesIcon />}
-            onClick={() => {
-              props.onClose();
-            }}
-          />
-        </CardTitle>
-
+        <CardHeader>
+          <Grid hasGutter>
+            <GridItem span={2}>
+              <img
+                style={{ maxHeight: '40px' }}
```

Comment:
[suggestion] It would be better to assign a static CSS class instead.

Reply:
Thanks, moved to separate scss file